docs/source/working_with_sources.rst

```diff
@@ -15,6 +15,8 @@ The following scenarios provide examples of how you would create a network, vcen
 .. _network:
 Creating a Network Source
 ^^^^^^^^^^^^^^^^^^^^^^^^^
+**IMPORTANT:** For a network scan, Quipucords must use the credentials to run some commands with sudo access. For more information about this requirement, see `Commands Used in Scans of Remote Network Assets <commands.html>`_
+
```

Comment:
same comment as above

Reply:
"sudo access" changed to "elevated privileges"